Automattic/wp-calypso
# Marketplace: Add test for plugin author page

#### Proposed Changes

* Add test to check plugin author page is correctly linked to prevent bugs like #66520.

#### Testing Instructions



* ~~Check out how to run e2e tests [here](https://github.com/Automattic/wp-calypso/tree/trunk/test/e2e#quick-start)~~
* ~~`yarn jest <path-to-calypso>/test/e2e/specs/plugins/plugins__developer.ts`~~
* `yarn test-client  -- <path-to-calypso>/client/my-sites/plugins/plugin-details-header`

#### Pre-merge Checklist






Fixes #66532
